ops_manual.md

```diff
@@ -5,30 +5,12 @@
 This is a tool developed within the Cyber Security tooling team to monitor estate-wide usage of the github security advisory service, specifically Dependabot.
 
 #### Architecture
-> TODO
 ```
-* A [Route53](https://aws.amazon.com/route53/) record pointing to ...
-* a [CloudFront](https://aws.amazon.com/cloudfront/) CDN which routes to ...
-* an [API Gateway](https://aws.amazon.com/api-gateway/) which routes to ...
-* some [Lambdas](https://aws.amazon.com/lambda/) which connect to ...
-* A [VPC](https://aws.amazon.com/vpc/) via an [ENI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html) which allow it to connect to ...
-* A PostgreSQL [RDS](https://aws.amazon.com/rds/) instance in the VPC.
+* A [Lambdas](https://aws.amazon.com/lambda/) which connect to GitHub and Splunk
```

Comment:
No need to the 'A' here and should we link the GitHub to Alphagov URL?

Reply:
Fixed